Applications/Games/The Sims 2/Local/script.js

```diff
@@ -0,0 +1,40 @@
+const LocalScript = include("engines.wine.quick_script.local_installer_script");
+const AppResource = include("utils.functions.apps.resources");
+const vcrun2010 = include("engines.wine.verbs.vcrun2010");
+const vcrun2013 = include("engines.wine.verbs.vcrun2013");
```

Comment:
I know that we currently don't enforce it, but please use the commonly used names for the included variables including the used upper and lower case letters, i.e. `Vcrun2013` instead of `vcrun2013`.

The reason for this is that it makes our work much easier if we later on discover that we need to do some refactoring on a verb. In such a situation it is likely that we search for all occurrences of `Vcrun2013`, which will not match similar but different strings like `vcrun2013`.